### ISSUE TITLE
1.3.5-beta settings dialogs have poor readability / accessibility

### ISSUE THREAD

**Issue by chrigrahcisco** [appium/appium-dot-app#456]
1.3.5-beta settings dialogs utilize grey text on a grey background. There is not enough contrast and it is difficult to read. Probably impossible for any sight impaired users.

**Comment by penguinho** [comment]
This is a Yosemite bug, I haven't figured out how to fix it. They show up correctly in interface builder. It's only for the checkboxes.

**Comment by penguinho** [comment]
If anyone can find anything about what Apple did in Yosemite to cause this, that would be helpful

**Comment by edgy360** [comment]
Yeah, I tried fixing this too back during the Yosemite developer
preview. There doesn't seem to be anything that we can do right now. I
even tried overriding the drawing and changing the colour, but didn't
get far with that.

**Comment by penguinho** [comment]
If you replace the checkbox text with NSLabels it does work, but that's annoying

**Comment by B-Dubs** [comment]
I'm not on Yosemite and still seeing this behavior.

**Comment by edgy360** [comment]
That’s interesting. I wonder if it is linked to the Target Platform SDK? Is there any chance that you could provide a screenshot and let us know what version you are using? Thanks.=

**Comment by B-Dubs** [comment]
![image](https://cloud.githubusercontent.com/assets/10745034/6006271/c81905de-aad6-11e4-942f-02ceadfef144.png)

I'm on OSX 10.9.5

**Comment by B-Dubs** [comment]
Plus, notice how Android settings come up when I select the Apple button.

**Comment by chrigrahcisco** [comment]
Mac OSX 10.10.2
Appium.app 1.3.5-beta

![screen shot 2015-02-02 at 1 32 38 pm](https://cloud.githubusercontent.com/assets/7501032/6006387/1be7485c-aae0-11e4-8c21-ac7488129d04.png)

**Comment by chrigrahcisco** [comment]
Mac OSX 10.10.2
Appium.app 1.3.4 production (still low contrast but not as bad)

![screen shot 2015-02-02 at 1 34 49 pm](https://cloud.githubusercontent.com/assets/7501032/6006421/555562ae-aae0-11e4-9855-eac53c255897.png)

**Comment by penguinho** [comment]
setting base sdk to 10.9 fixes it. I just have to fix one ARC conversion issue to make this work

**Comment by Machx** [comment]
This seems to still be occurring in the official 1.3.5 release as I am seeing this for macs running 10.9.x

**Comment by B-Dubs** [comment]
Yeah, I just upgraded to Appium 1.3.5 and still see the iOS Settings looking like this on OSX 10.9.5...
![image](https://cloud.githubusercontent.com/assets/10745034/6155420/dedb9a56-b1f8-11e4-9dfd-20c1cc456088.png)

**Comment by penguinho** [comment]
I'm wondering, why it is black. I don't think I set the type to HUD, but I'll have to double check that.

**Comment by B-Dubs** [comment]
Yeah, it is odd.  Unless it somehow is related to settings local to my machine?

**Comment by edgy360** [comment]
Reports of another user experiencing the same issue: https://twitter.com/HolyEggplant/status/565857524801634304

**Comment by rrenzetti** [comment]
I just installed 1.3.5 and I'm also seeing this issue. My OS version is 10.9.4

**Comment by jmkristian** [comment]
I see this problem in Appium.app versions 1.3.5, 1.3.6 and 1.3.7, with MacOS version 10.9.4.
![illegible](https://cloud.githubusercontent.com/assets/182744/6987104/d2fc5612-d9f8-11e4-96f8-06274f0fd655.png)

**Comment by aaiii** [comment]
Is there a fix or work around to this issue?
